src/ui/server/controllers/insurance/policy-and-export/map-submitted-data/policy-contact/index.ts

```diff
@@ -47,7 +48,7 @@ const mapSubmittedData = (formBody: RequestBody, application: Application): obje
 
     delete populatedData[NAME];
 
-    if (populatedData[POSITION]) {
+    if (populatedData[POSITION] && isPolicyContactChanged(application.policyContact, formBody as ApplicationPolicyContact)) {
```

Comment:
Can we split this up bit hard to read, maybe move `isPolicyContactChanged` to a line above, `const policyContactHasChanged = `

Should the new helper be called `hasPolicyContactChanged`?